web_app/alembic/versions/a009512f5362_start_price_field_added_in_position.py

```diff
@@ -30,13 +30,15 @@ def upgrade() -> None:
     if column_exists('position', 'start_price'):
         print("Column 'start_price' already exists, skipping creation.")
     else:
-        op.add_column('position', sa.Column('start_price', sa.DECIMAL(), nullable=False))
+        op.add_column('position', sa.Column('start_price', sa.DECIMAL(), nullable=False,
```

Comment:
Why did you change old migration?

Reply:
without this config  its giving error

Reply:
which error, please ,be more specific, if you have any error, attach it here with explanation what did you run the command to get this error and traceback of this error